scripts/new-branch.sh

```
@@ -1,104 +1,104 @@
 fi
 
 TYPE=$1
 DESCRIPTION=$2
 
-# Types valides
+# Valid types
 VALID_TYPES=("feat" "fix" "chore" "docs" "refactor" "test")
 
-# Vérifier que le type est valide
+# Verify type is valid
 if [[ ! " ${VALID_TYPES[@]} " =~ " ${TYPE} " ]]; then
```

Shellcheck (0.10.0)

[error] Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).

(SC2199)